Python/tigre/utilities/io/NikonDataLoader.py

```diff
@@ -157,9 +158,11 @@ def loadNikonProjections(folder, geometry, angles, **kwargs):
     image = numpy.asarray(image).astype(numpy.float32)
     projections = numpy.zeros([len(indices), image.shape[0], image.shape[1]], dtype=numpy.single)
     projections[0, :, :] = -numpy.log(image / float(geometry.whitelevel))
-    index = 1
+    #Python uses zero-based indexing
```

Comment:
Yes, but we just read the "zeroeth" image in the previous line! so we can skip reading it!

Reply:
Sure, that's fine, but in the previous version, `index `eventually ends up one larger than the size of `projections`

You could skip the first projections with
`for index, i in enumerate(tqdm(indices[1:]),1):`

Reply:
Did it? Weird, I thought I had this well tested. 
Sure the change you suggest make sense. Mind making a commit? then I'll merge the PR.